packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -72,7 +72,7 @@ export class AtomicSegmentedFacet
    * The number of values to request for this facet.
    * Also determines the number of additional values to request each time more values are shown.
    */
-  @Prop({reflect: true}) public numberOfValues = 6;
+  @Prop({reflect: true}) public numberOfValues = 8;
```

Comment:
Johnny recommended 6 if I recall correctly, why switch it to 8?

Reply:
I was just testing out how 8 facets looked